src/fr24/history.py

```diff
@@ -287,10 +288,11 @@ def playback_track_ems_dict(point: TrackData) -> PlaybackTrackEMSRecord | None:
     return None
 
 
-# TODO: add ems, metadata.
 def playback_df(result: Playback) -> pd.DataFrame:
     """
     Transform each point in the flight track to a pandas DataFrame.
+
+    Note: EMS and metadata will not be serialised.
```

Comment:
😢

Reply:
Fixed in https://github.com/cathaypacific8747/fr24/pull/24/commits/3edf4477455e432cc4ad89727912f63012cd16c3